glass/lensing.py

```diff
@@ -67,6 +72,8 @@ def from_convergence(
         Maximum angular mode number to use in the transform.
     potential, deflection, shear : bool, optional
         Which lensing maps to return.
+    discretized : bool
+        Whether to apply a pixel window function to the shear.
```

Comment:
@ntessore this was missing, I've had a crack. Please feel free to edit.

Reply:
Close!
```suggestion
    discretized : bool
        Correct the pixel window function in output maps.
```